versioned_docs/version-latest/protocol/pulsar.mdx

```diff
@@ -469,7 +404,111 @@ Initial plotting for plot: #0 (/home/username/.local/share/pulsar/farms)
 
 That's it! Enjoy and Happy Farming!
 
-#### Moving the Farming Process to the Background
+
+## Troubleshooting
```

Comment:
Love the idea of moving all this stuff to the section at the bottom to keep the quick start simpler 🤟